bindings-go/apis/v2/componentdescriptor.go

```diff
@@ -427,3 +437,35 @@ func (o *ComponentReference) GetIdentity() Identity {
 func (o *ComponentReference) GetIdentityDigest() []byte {
 	return o.GetIdentity().Digest()
 }
+
+// DigestSpec defines the digest and algorithm.
+// +k8s:deepcopy-gen=true
+// +k8s:openapi-gen=true
+type DigestSpec struct {
+	Algorithm string `json:"algorithm"`
+	Value     string `json:"value"`
+}
+
+// SignatureSpec defines the signature and algorithm.
+// +k8s:deepcopy-gen=true
+// +k8s:openapi-gen=true
+type SignatureSpec struct {
```

Comment:
we should have an optional field with a an expiration date.
We have different use cases where the signature is only valid for a certain amount of time. We should also reflect that in the struct.